### DISCUSSION TITLE
can't detect button icons

### DISCUSSION THREAD

**Discussion by nobaraos12** [JulianPrieber/littlelink-custom#238] [In progress]
I've noticed littlelink can't detect google drive and simplelogin button icons

**Comment by JulianPrieber** [comment]
Could you be more specific with your issue?

**Comment by nobaraos12** [comment]
> Could you be more specific with your issue?

when added the url of an google drive document little link doesn't detect the google drive icon, and for simplelink it doesn't detect the icon from the website

**Comment by JulianPrieber** [comment]
I'm moving this to in progress.

Still have to do some testing, but I think this is fixable.
But don't count on that this will be fixed any time soon.

I might be able to add a last check to the get function that retrieves the favicon from the base domain if none is found.
